Formula/h/hurl.rb

```diff
@@ -6,6 +6,11 @@ class Hurl < Formula
   license "Apache-2.0"
   head "https://github.com/Orange-OpenSource/hurl.git", branch: "master"
 
+  livecheck do
```

Comment:
```suggestion
  # Upstream uses GitHub releases to indicate that a version is released
  # (there's also sometimes a notable gap between when a version is tagged and
  # and the release is created), so the `GithubLatest` strategy is necessary.
  livecheck do
```

The formula's `stable` URL is a tag archive (not a GitHub release asset), so we need to document why `GithubLatest` is used here.

The suggestion above is probably the closest explanation of this situation out of our typical explanatory comments. It may not be 100% true for every release but that's how it goes.